projects/hslayers/src/components/draw/draw-layer-metadata.component.ts

```diff
@@ -126,5 +125,8 @@ export class HsDrawLayerMetadataDialogComponent
 
   selectLayer(layer): void {
     this.data.selectLayer(layer);
+    setTimeout(() => {
```

Comment:
Why do we need this timeout and dialog closure after selecting a layer?

Reply:
We dont, it just felt better as it takes some time to get layer trom Layman

Reply:
It would be better to remove it, because the layer will get loaded anyways. What the timeout does, it gives enough time for user to click the layer second time and two identical layers gets added to the map.

Reply:
Sure sounds reasonable , thanks :)